pkg/ccl/crosscluster/logical/dead_letter_queue.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2024 The Cockroach Authors.
```

Comment:
I'd remove the release note from this, as LDR is in private preview is undocumented for a few months.